2022 - Horus Heresy.gst

```diff
@@ -1350,6 +1350,24 @@ If a Skimmer is Immobilised or Wrecked, its base is removed, if possible. If thi
     </categoryEntry>
     <categoryEntry name="Blackshields Primary Detachment" id="3410-6cb0-2f2b-a28a" hidden="false"/>
     <categoryEntry name="Blackshields Optional Detachment" id="e3b2-7cd8-eab7-92eb" hidden="false"/>
+    <categoryEntry name="Dark Angels - Crusade Detachment" id="df67-4304-4f8a-b4ea" hidden="false"/>
```

Comment:
I like this implementation. So you add this if there's a crusade force org of the appropriate type, and select it in solar if true.